### ISSUE TITLE
Lock export settings after starting the workload

### ISSUE THREAD

**Issue by o1230sponge** [cirquit/trdrop#85]
While you're exporting, it's currently possible to toggle the overlay options and cause some of your frames to show game video and some not show game video. Making this option locked after you start the encode seems like it wouldn't be a hard feature to add, and prevent any accidental clicks from messing up any progress.

**Comment by cirquit** [comment]
Yes, this is a good proposal, I'll envision trdrop to be also an "exploratory" tool as well, that's why I kept everything modifiable. But making such a "lock" should be easily done.
